ts/utils/remoteContextualHelp.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { View } from "native-base";
+import { ContextualHelpProps } from "../components/screens/BaseScreenComponent";
+
+/**
+ * Create the object needed to ensure that the Contextual Help question mark is visible
+ * when the Contextual Help is filled remotely.
+ *
+ */
+export const remoteContextualHelp = (): ContextualHelpProps => ({
```

Comment:
I guess we don't need to create a new object each time. It's always the same.
A const could be fine

```typescript
export const emptyContextualHelp: ContextualHelpProps = {
  title: "",
  body: () => <View />
};
```

Could be `emptyContextualHelp` (same for the filename) a more descriptive name? It hasn't any meanings about "remote"